### DISCUSSION TITLE
CSV Tall vs Wide

### DISCUSSION THREAD

**Discussion by mvantino** [CMSgov/hospital-price-transparency#37] [Q&A] [answered]
I am just getting my feet wet on this project and am not understanding the difference between CSV Tall vs CSV Wide, so I'm unclear which to pick.  Is there something here that details what the actual differences are?  The examples appear pretty similar.

**Comment by RidenourK** [comment, marked as answer]
@mvantino. Thank you for your question. 

As indicated in the CY 2024 OPPS/ASC Final Rule, the “tall’’ format contains separate payer and plan data elements that provide the benefit of static header naming with less opportunity for standardization error and that is similar to existing output files that many hospitals have been using to build their MRFs. The ‘‘wide’’ format contains variable payer-specific negotiated charge data elements that incorporate the payer and plan name into a single column header, which may reduce the file size because many data elements would not need to be repeated as frequently. Please refer to the discussion at [88 FR 82088](https://www.federalregister.gov/d/2023-24293/p-4893).
